features/support/env.rb

```diff
@@ -7,6 +7,10 @@
 require 'cucumber/rails'
 require 'database_cleaner'
 
+require 'capybara/poltergeist'
+Capybara.javascript_driver = :poltergeist
+WebMock.allow_net_connect!
```

Comment:
I'd like to generally disable net connect with the exception of specific exemptions, but we can make that change later